src/components/app.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+import React from 'react';
+import SearchForm from './redditSearch/search';
+import SearchResultList from './redditSearch/resultList';
+import superagent from 'superagent';
+
+import '../style/app.scss';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      results: [],
+      error: false,
+    };
+
+    this.searchReddit = this.searchReddit.bind(this);
+    this.fetchData = this.fetchData.bind(this);
+  }
+
+  searchReddit(search, limit) {
+    let url = `https://www.reddit.com/r/${search}.json?limit=${limit}`;
+
+    return this.fetchData(url)
+      .then(posts => {
+        console.log('fetching data');

```

Comment:
WHAT IS THIS?!  WHY?! WHY DO YOU DO THIS TO ME?!

Just kidding. But delete the console.log().